modules/core/include/opencv2/core/opencl/opencl_info.hpp

```diff
@@ -94,15 +111,10 @@ static void dumpOpenCLInformation()
 
         DUMP_MESSAGE_STDOUT("Current OpenCL device: ");
 
-#if 0
-        DUMP_MESSAGE_STDOUT("    Platform = " << device.getPlatform().name());
-        DUMP_CONFIG_PROPERTY("cv_ocl_current_platformName", device.getPlatform().name());
-#endif
-
-        const char* deviceTypeStr = (device.type() == Device::TYPE_CPU) ? "CPU" :
-            (device.type() == Device::TYPE_GPU ? device.hostUnifiedMemory() ? "iGPU" : "dGPU" : "unknown");
+        String deviceTypeStr = getDeviceTypeString(
+            device.type(), device.hostUnifiedMemory());
         DUMP_MESSAGE_STDOUT("    Type = " << deviceTypeStr);
-        DUMP_CONFIG_PROPERTY("cv_ocl_current_deviceType", deviceTypeStr);
+        DUMP_CONFIG_PROPERTY("cv_ocl_current_deviceType", deviceTypeStr.c_str());
```

Comment:
unnecessary change

Reply:
In this function, there is no cv::format as in line 103, right.